source/tutorials/monitor-minio-tenant.rst

```diff
@@ -62,6 +62,20 @@ The command returns a bearer token similar to the following:
      static_configs:
      - targets: ['HOSTNAME:9000']
 
+If using a self signed certificate, add the following to skip verification.
```

Comment:
```suggestion
Prometheus by default validates the TLS certificate presented by the MinIO 
Tenant. You can modify the `scrape_config` to explicitly point the Prometheus 
service at the Certificate Authority (CA) to use for validating the MinIO Tenant 
certificate, *or* disable TLS verification entirely.

MinIO Pods use SNI to determine which TLS certificate to respond with during the 
TLS handshake. Specifically, the ``static_configs.targets`` hostname determines 
the TLS certificate the MinIO Tenant responds with:

- If using the MinIO Kubernetes Service DNS name or Cluster IP address as the 
  scrape target, specify the path to the Kubernetes cluster CA:

  .. code-block:: shell
     :class: copyable
     
     scrape_config:
     - job_name: minio-job
       ...
       tls_config:
          ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
          
  If the Prometheus service is *not* running as a pod in the Kubernetes 
  cluster, you can copy the CA file from any pod in the cluster onto the 
  Prometheus service host. 
  
- If using a DNS name associated to a custom certificate configured
  using the  :kubeconf:`spec.externalCertSecret` configuration setting, 
  specify the path to the CA used to sign that certificate:

  .. code-block:: shell
     :class: copyable
     
     scrape_config:
     - job_name: minio-job
       ...
       tls_config:
          ca_file: /path/to/minio-ca.crt
 
- To disable TLS verification for the MinIO scrape job, specify 
  ``tls_config.insecure_skip_verify: true``
  
  
  .. code-block:: shell
     :class: copyable
     
     scrape_config:
     - job_name: minio-job
       ...
       tls_config:
         insecure_skip_verify: true
         
  Disabling TLS verification may increase the risk of certain attacks or 
  exploits associated to unverified TLS connections. 
```

Reply:
cc @kerneltime @harshavardhana 

This might also be a better fit under step 2, since that's the part associated to the Prometheus config itself.